src/lib.rs

```diff
@@ -146,21 +151,13 @@ where
         result
     }
 
-    fn execute(&mut self, query: &str) -> QueryResult<usize> {
-        let id = UniqueId::new();
-        probes::query__start!(|| (&id, self.id, query));
-        let result = self.inner.execute(query);
-        probes::query__done!(|| (&id, self.id));
-        result
```

Comment:
These probes are pretty important :). Did they get replaced somewhere I'm missing?

Reply:
Based on my light reading of the new 2.0.0-rc source, the `load` method is the closest alternative, although I remember that not all queries actually went through that method when we first implemented this crate.

Reply:
Hmm, reading more, it seems that the `execute` method is gone entirely. Is there any documentation that describes this as the method through which all queries pass? Seems like it must be, but I'm probably missing something.

Reply:
I believe the calls have all been consolidated into `load` and `execute_returning_count`, both part of the new [Connection](https://docs.diesel.rs/master/diesel/connection/trait.Connection.html) API. Both of those have probes already though.

Reply:
Agreed seems like everything must go through there. I thought there may have been some hidden method like in the older versions. Thanks!